packages/snaps-sdk/src/types/methods/get-background-events.ts

```diff
@@ -0,0 +1,18 @@
+import type { Json } from '@metamask/utils';
+
+import type { SnapId } from '../snap';
+
```

Comment:
We normally do something like this if no params are expected.
```suggestion

export type GetBackgroundEventsParams = never;

```